### PR TITLE
Aj/support staking candidate jailed

### PR DESCRIPTION
**Pull request by webbushka**
### Improvements

- Allow staking on Jailed Validator
- Support staking on both Jailed and Candidate
- Change line length to 100, the previous length was too long

### Bug Fixes

- Default the users hash amount to 0

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+
+### Improvements
+
+- Change line length to 100, the previous lentgh was too long
+
+### Bug Fixes
+
+- Removed staking from Jailed Validator
+- Default the users hash amount to 0
+
```
Comment (crussell-figure) [inline CHANGELOG.md:22]: these are all 1.2.1 as 1.2.1 hasnt been released yet

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/provenance-io/explorer-frontend"
   },
-  "version": "1.2.0",
+  "version": "1.2.2",
```
Comment (crussell-figure) [inline package.json:10]: 1.2.1

**Review by crussell-figure — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (webbushka) [reply CHANGELOG.md:22]: yeah... I tagged the last PR with 1.2.1, I could remove the tag but then git history will be weird. So I'd rather release 1.2.2 and just have 1.2.1 never actually get released.

Comment (crussell-figure) [reply CHANGELOG.md:22]: lol kk

**Review by crussell-figure — APPROVED** [review]

**Pull request merged**